src/Uno.UI/Microsoft/UI/Xaml/Controls/CalendarView/CalendarView_Partial.cs

```diff
@@ -0,0 +1,2724 @@
+					newDisplayMode = (CalendarViewDisplayMode)args.NewValue;
+
+					OnDisplayModeChanged(
+						(CalendarViewDisplayMode)(oldDisplayMode),
+						(CalendarViewDisplayMode)(newDisplayMode)
+					);
+				}
+					break;
+				case DependencyProperty CalendarView_IsTodayHighlightedProperty when CalendarView_IsTodayHighlightedProperty == CalendarView.IsTodayHighlightedProperty:
+					OnIsTodayHighlightedPropertyChanged();
+					break;
+				case DependencyProperty CalendarView_IsGroupLabelVisibleProperty when CalendarView_IsGroupLabelVisibleProperty == CalendarView.IsGroupLabelVisibleProperty:
+					OnIsLabelVisibleChanged();
+					break;
+
+				// To reduce memory usage, we move lots font/brush properties from CalendarViewItem to CalendarView,
+				// the cost is we can't benefit from property system to invalidate measure/render automatically.
+				// However changing these font/brush properties is not a frequent scenario. So once they are changed
+				// we'll manually update the items.
+				// Basically we should only update those affected items (e.g. when PressedBackground changed, we should only update
+				// the item which is being pressed) but to make the code simple we'll update all realized item, unless
+				// we see performance issue here.
+
+				// Border brushes and Background (they are chromes) will take effect in next Render walk.
+				case DependencyProperty CalendarView_FocusBorderBrushProperty when CalendarView_FocusBorderBrushProperty == CalendarView.FocusBorderBrushProperty:
+				case DependencyProperty CalendarView_SelectedHoverBorderBrushProperty when CalendarView_SelectedHoverBorderBrushProperty == CalendarView.SelectedHoverBorderBrushProperty:
+				case DependencyProperty CalendarView_SelectedPressedBorderBrushProperty when CalendarView_SelectedPressedBorderBrushProperty == CalendarView.SelectedPressedBorderBrushProperty:
+				case DependencyProperty CalendarView_SelectedBorderBrushProperty when CalendarView_SelectedBorderBrushProperty == CalendarView.SelectedBorderBrushProperty:
+				case DependencyProperty CalendarView_HoverBorderBrushProperty when CalendarView_HoverBorderBrushProperty == CalendarView.HoverBorderBrushProperty:
+				case DependencyProperty CalendarView_PressedBorderBrushProperty when CalendarView_PressedBorderBrushProperty == CalendarView.PressedBorderBrushProperty:
+				case DependencyProperty CalendarView_CalendarItemBorderBrushProperty when CalendarView_CalendarItemBorderBrushProperty == CalendarView.CalendarItemBorderBrushProperty:
+				case DependencyProperty CalendarView_OutOfScopeBackgroundProperty when CalendarView_OutOfScopeBackgroundProperty == CalendarView.OutOfScopeBackgroundProperty:
+				case DependencyProperty CalendarView_CalendarItemBackgroundProperty when CalendarView_CalendarItemBackgroundProperty == CalendarView.CalendarItemBackgroundProperty:
+					ForeachHost(pHost => 
+					{
+						ForeachChildInPanel(
+							pHost.Panel,
+							pItem =>
+							{
+								pItem.InvalidateRender();
```

Comment:
Potential null-ref

Reply:
@kazo0 does all those possible null-ref are affecting you or they are just code analysis ? They are pretty common in code imported from winUI and mainly due to not-so-strong typing of C++ but are actually impossible ... or are usually underlying another deeper issue.

Reply:
All the ones with the call to `ForeachChildInPanel` are affecting me. It causes a null-ref when using the `CalendateDatePicker` with a custom `CalendarViewStyle` on non Windows platforms.

Reply:
This has been fixed by this commit https://github.com/unoplatform/uno/pull/6098/commits/e753025c3b5906df668e8215fc2d3ce108f9a31f